src/main/java/core/basesyntax/exception/NoDaoAnnotationException.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.exception;
+
+public class NoDaoAnnotationException extends RuntimeException {
```

Comment:
Maybe will be better set name of exception class more global: like NoAnnotationException? And then we can use this exception for any annotations